google/cloud/bigquery/_helpers.py

```diff
@@ -319,6 +319,7 @@ def _timestamp_to_json_row(value):
     """
     if isinstance(value, datetime.datetime):
         value = _microseconds_from_datetime(value) * 1e-6
+        value = round(value, 6)
```

Comment:
I'm curious how this does anything. Doesn't `_microseconds_from_datetime` return an integer?

Reply:
Yes, it returns an integer as we can see here https://github.com/googleapis/python-cloud-core/blob/5768ce33a182c97d9c3b2cd77d2fec6a39326b68/google/cloud/_helpers.py#L215